### PR TITLE
ja: update glossary link description

### PR DESCRIPTION
**Pull request by PassionPenguin**
mdn/content#34454. just a replacement given not all entries under `ja/glossary` are up-to-date

### CODE REVIEW

File: files/ja/glossary/breadcrumb/index.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ l10n:
 
 サイト内での位置を表すパンくずリストは、この記事なら概ね次のようなかたちになるでしょう。
 
-[MDN](/) > [用語集](/ja/docs/Glossary) > パンくずリスト
+[MDN](/) > 関連用語 > パンくずリスト
```
Comment (mfuji09) [inline files/ja/glossary/breadcrumb/index.md:14]: This modifying seems incorrect.
Please compare this with en-US version.

**Review by mfuji09 — CHANGES_REQUESTED** [review]
Please use half-width colon ":" instead of full-width colon "：". It is by a custom of ja community.

And please check another comment.

**Comment by PassionPenguin** [comment]
> Please use half-width colon ":" instead of full-width colon "：". It is by a custom of ja community.
> 
> And please check another comment.

sorry for my mistake! they should now looks good :)

**Review by mfuji09 — APPROVED** [review]
LGTM.

**Pull request merged**